packages/webcert/src/feature/certificate/CertificateSidePanel/Footer/SidePanelFooter.tsx

```diff
@@ -13,14 +13,12 @@ const TextWrapper = styled.div`
 `
 
 interface Props {
-  backgroundColor: 'iu-bg-grey-300' | 'iu-bg-main' | 'iu-bg-white'
-  textColor?: 'iu-color-text' | 'iu-color-white' | 'iu-color-main'
   additionalStyles?: string
 }
 
-const SidePanelFooter: React.FC<Props> = ({ children, backgroundColor, textColor, additionalStyles }) => {
+const SidePanelFooter: React.FC<Props> = ({ children, additionalStyles }) => {
   return (
-    <Root className={`${backgroundColor} ${textColor} ${additionalStyles} iu-border-grey-300`}>
+    <Root className={`iu-bg-muted-light iu-color-secondary-dark ${additionalStyles} iu-border-grey-300`}>
```

Comment:
It doesn't change the functionality or the readability of the code to change the name of the attribute. Since this code doesn't touch the attribute additionalStyles I will leave it as it is.